### PR TITLE
Build: Generate files `d.ts`

### PR DESCRIPTION
**Pull request by sarvaje**
Related #432

### CODE REVIEW

File: tests/helpers/test-server.ts [hunk 1]
```diff
@@ -166,10 +166,8 @@ class Server {
 }
 
 /** Returns a testing server */
-const createServer = (isHTTPS?: boolean) => {
-    const server = new Server(isHTTPS);
+export const createServer = (isHTTPS?: boolean) => {
+    const a = new Server(isHTTPS);
```
Comment (molant) [inline tests/helpers/test-server.ts]: What about doing just a `return new Server(isHTTPS);`? Not a huge fan of `a`

**Comment by molant** [comment]
@alrra, @sarvar is the `.d.ts` file going to be included directly on the npm package with this configuration or is there something else that needs to be done?

File: tests/helpers/test-server.ts (hunk 1 above)
Comment (sarvaje) [inline tests/helpers/test-server.ts]: Ups, I forgot to restore the name to `server`... but yes, maybe is better just do a return.

Comment (sarvaje) [inline tests/helpers/test-server.ts]: @molant done!

**Pull request merged**